### ISSUE TITLE
Any-to-Any Generation via Composable Diffusion

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#217]
# TL; DR

- Jointly embed conditions from multiple modalities through contrastive learning, and design multimodal generation with cross-attention-reinforced latent diffusion model.

# Motivation

- **Any-to-Any** Generation. (image, text, audio, video)

# Formulation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/ff3f9c9e-5dff-4b66-9d9b-df2ef9059c6d)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/f89b525e-670b-4fa0-8442-ddea18cc8175)

- First train modality-specific condition encoders with contrastive learning.
- Then train single-modality diffusers.
- Finally train environment encoders and cross-modal cross-attention layers for multimodal joint generation.

![image](https://github.com/reyllama/paper-reviews/assets/46308407/0c43c8c6-791d-4f63-bab7-4145b12642e6)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/4402cd40-fde2-457b-9f0d-6d032efdcdf6)

# Evaluation

![image](https://github.com/reyllama/paper-reviews/assets/46308407/98acf439-76ac-40a2-8a6d-ca3875cc1f29)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/0376a46d-3c89-468a-a006-c4b30d5eb5ce)
![image](https://github.com/reyllama/paper-reviews/assets/46308407/31a14bf5-1068-4e4d-8056-b342987cf783)

# Inspiration

- Ideas on publically open multimodal datasets.
- Contrastive learning is mighty.
- Ideas on video LDM.
